config.yml

```diff
@@ -0,0 +1,77 @@
+dataFileName: 'pattern.yml'
+prototyperDir: './patternlab'
+patternlab:
+  dest: './patternlab'
```

Comment:
This was built from the current version. Next Generator version will remove this distinction.